build.sbt

```diff
@@ -2,17 +2,30 @@ import Settings._
 import Testing._
 
 
+lazy val rawlsUtil = project.in(file("util"))
```

Comment:
do we want to think about not calling these rawlsFoo in favour of workbenchFoo or something?